### PR TITLE
Show error pages content in normal layout

### PR DESCRIPTION
**Pull request by salbertson**
This change gives us more control over the look of error pages.

http://wearestac.com/blog/dynamic-error-pages-in-rails

https://github.com/thoughtbot/hound/issues/515

### CODE REVIEW

**Comment by salbertson** [comment]
https://github.com/thoughtbot/hound/issues/515

**Comment by arunagw** [comment]
Looks ok to me. I have never done any customization for error page via rails. 

For example will it serve 500 if DB is down ?

**Comment by srt32** [comment]
Would it be possible to add a spec around the error page routing?

**Comment by teoljungberg** [comment]
I would like to see a request spec for this aswell

File: config/routes.rb [hunk 1]
```diff
@@ -20,5 +20,9 @@
   resources :repo_syncs, only: [:index, :create]
   resource :user, only: [:show]
 
+  %w( 404 422 500 ).each do |status_code|
```
Comment (gylaz) [inline config/routes.rb]: no spaces after `(` and before `)`.

**Comment by salbertson** [comment]
@srt32 @teoljungberg good call on tests, done. I added a simple test that makes sure error pages render correctly and avoided testing the way Rails handles errors.

File: config/routes.rb (hunk 1 above)
Comment (salbertson) [inline config/routes.rb]: Why didn't Hound comment?

**Pull request merged**